### PR TITLE
Refactor updateAnnotations and reformat logs and errors.

### PR DESCRIPTION
**Pull request by alejandroEsc**
Cleaning up logs, fixing small errors in code, moving logic around to be a bit more readable.

### CODE REVIEW

File: cloud/ssh/actuators/machine/actuator_status.go [hunk 1]
```diff
@@ -104,6 +104,10 @@ func (a *Actuator) setMachineStatus(machine *clusterv1.Machine, status MachineSt
 }
 
 func (a *Actuator) updateAnnotations(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
+	if a.v1Alpha1Client != nil {
```
[the remark this answers is not in the document]
Comment (alejandroEsc) [reply cloud/ssh/actuators/machine/actuator_status.go]: was just about to fix that.

File: cloud/ssh/actuators/machine/actuator.go [hunk 2]
```diff
@@ -205,112 +195,92 @@ func (a *Actuator) Delete(c *clusterv1.Cluster, m *clusterv1.Machine) error {
...
 }
 
 // Update updates a machine and is invoked by the Machine Controller
-func (a *Actuator) Update(cluster *clusterv1.Cluster, goalMachine *clusterv1.Machine) error {
-	goalMachineName := goalMachine.ObjectMeta.Name
-	clusterName := cluster.ObjectMeta.Name
-	glog.Infof("Updating Machine %v for cluster %v.", goalMachineName, clusterName)
+func (a *Actuator) Update(c *clusterv1.Cluster, goalMachine *clusterv1.Machine) error {
+	goalMachineName := goalMachine.Name
+	glog.Infof("Updating Machine %s for cluster %s.", goalMachineName, c.Name)
```
[the remark this answers is not in the document]
Comment (alejandroEsc) [reply cloud/ssh/actuators/machine/actuator.go]: Actually its done on purpose, look at the first log messages given in each case. Now i dont understand the second comment?

[the remark this answers is not in the document]
Comment (davidewatson) [reply cloud/ssh/actuators/machine/actuator.go]: Sorry.

I meant that I think the previous log messages were preferrable. Within context it is almost always clear which operation is involved. When the characters `Creating - ` occur at the begining of each line it makes it more difficult for humans to read. The format is not clearly easier for a computer to parse either with dash, colon and comma separators.

If you think the log messages could be formated better please submit a seperate PR containing only those changes and the rational.

[the remark this answers is not in the document]
Comment (alejandroEsc) [reply cloud/ssh/actuators/machine/actuator.go]: Hmm, sure i can revert and do things like others, no worries. The truth is that the logs as things fly dont always appear in sequence, and as such sometimes its difficult to make out what logs belong together in sequence. There are many ways to solve that, even using some session id, or processid, which probably would be over kill. Ill revert the logs but clean them up.

**Comment by alejandroEsc** [comment]
There are actually a few bug fixes, issues with errors, golang style issues etc.. addressed here. Delete was not the only thing fixed/addressed here.

**Pull request merged**